docs/api.md

```diff
@@ -1310,25 +1310,31 @@ Generate a bash completion script. Users of your application can install this
 script in their `.bashrc`, and yargs will provide completion shortcuts for
 commands and options.
 
-.showHelp(consoleLevel='error')
+.showHelp([consoleLevel | printCallback])
 ---------------------------
 
-Print the usage data using the [`console`](https://nodejs.org/api/console.html) function `consoleLevel` for printing.
+Print the usage data.
 
-Example:
```

Comment:
Shouldn't we keep `Example:` before examples?

Reply:
Right now there is an example for every option - no args, string arg, and function arg. Would you like me to move the examples to separate section after all the text?

Reply:
No, that's okay. Most examples in the page does seem to have this anyway.